packages/react-native-reanimated/src/runtimes.ts

```diff
@@ -35,11 +36,16 @@ export function createWorkletRuntime(
   name: string,
   initializer?: WorkletFunction<[], void>
 ): WorkletRuntime {
+  // Assign to a different variable as __reanimatedLoggerConfig is not captured
+  // identifier on the Worklet runtime.
+  const config = __reanimatedLoggerConfig;
+
   return NativeReanimatedModule.createWorkletRuntime(
     name,
     makeShareableCloneRecursive(() => {
       'worklet';
       registerReanimatedError();
+      registerLoggerConfig(config);
```

Comment:
I think this will be enough. The `configureReanimatedLogger` function should be called called in the global react scope before calling `createWorkletRuntime` so changes made to the config will be also visible in other runtimes when they are created.

We can add to docs information how to use `configureReanimatedLogger` properly (call it before using anything else from reanimated).